### PR TITLE
Closes #41 - Adding anti-forgery support to distributed invocations

### LINKED ISSUE #41: Message Bus should support anti-forgery
With the new [lattice](https://github.com/wascc/wascc-host/pull/38) feature, invocations will be sent across the wire other hosts. While TLS can be used on the endpoints and authentication can be used to limit access, we should still be able to prove that an invocation came from the host from which it claims to have originated.

## Bad Actor Scenario

In keeping with defence-in-depth, you can't assume all of your layers of protection are working, so you need to defend everywhere you can. In this case, assume someone has managed to gain access to a NATS server or a NATS leaf node being used by the lattice distributed bus, and has stolen enough credentials so that they can legitimately publish messages to the lattice RPC prefix (`wasmbus.*`). They could then generate an invocation that might allow data exfiltration, corruption, or perform DDoS attacks. If every recipient in the lattice verifies that the invocation was signed by the host identity in the payload, we can _at least_ verify that the sender has lattice credentials _and_ is using our key system. Put another way, we would force the impersonated waSCC host to uniquely identify itself in order to send seemingly legitimate invocations. 

These safeguards squarely put this bad actor scenario in the "far too much effort to be worth it" category.

Finally, if the above scenario were to take place, we would then gain the ability to place an offending impersonated host on a `deny list`. If we become aware of a forger on the lattice, we can block it immediately.

### PR DESCRIPTION
**Pull request by autodidaddict**

### CODE REVIEW

File: src/inthost.rs [hunk 1]
```diff
@@ -186,30 +202,124 @@ pub(crate) fn gen_remove_actor(msg: Vec<u8>, binding: &str, capid: &str) -> Invo
...
 }
 
+impl WasccEntity {
+    pub fn url(&self) -> String {
+        match self {
+            WasccEntity::Actor(pk) => format!("{}://{}", bus::URL_SCHEME, pk),
+            WasccEntity::Capability { capid, binding } => format!(
+                "{}://{}/{}",
+                bus::URL_SCHEME,
+                capid.replace(":", "/").replace(" ", "_").to_lowercase(),
```
Comment (ewbankkit) [inline src/inthost.rs:229]: Can use `normalize_capid()` from `src/bus/mod.rs`?

Comment (autodidaddict) [reply src/inthost.rs:229]: The bus normalize converts the colons into dots, and this conversion makes them slashes and more url-like.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,23 @@ All notable changes to this project will be documented in this file.
...
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.10.0] - In Progress
+
+This release includes several lattice-related enhancements as well as some security and stability improvements.
+
+### Changed
+
+* The `Invocation` type now includes its own set of claims that must be verified by receiving code. This prevents invocations from being forged on the wire in the case of intrusion.
+
```
Comment (ewbankkit) [inline CHANGELOG.md:16]: Also note that `InvocationTarget` has been renamed to `WasccEntity`.

File: src/inthost.rs (hunk 1 above)
Comment (ewbankkit) [reply src/inthost.rs:229]: OK.

**Review by ewbankkit — APPROVED** [review]
LGTM (modulo minor comments).

**Pull request merged**